GooglePay/src/test/java/com/braintreepayments/api/googlepay/GooglePayLauncherUnitTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.braintreepayments.api.googlepay
+
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.ActivityResultRegistry
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.fragment.app.FragmentActivity
+import com.google.android.gms.wallet.PaymentDataRequest
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class GooglePayLauncherUnitTest {
+
+    private val activityResultLauncher: ActivityResultLauncher<GooglePayPaymentAuthRequestParams?> =
+        mockk<ActivityResultLauncher<GooglePayPaymentAuthRequestParams?>>(relaxed = true)
+    private val callback: GooglePayLauncherCallback? = mockk<GooglePayLauncherCallback>(relaxed = true)
+    private val activityResultRegistry: ActivityResultRegistry? =
+        mockk<ActivityResultRegistry>(relaxed = true)
```

Comment:
```suggestion
    private val activityResultLauncher = mockk<ActivityResultLauncher<GooglePayPaymentAuthRequestParams?>>(relaxed = true)
    private val callback = mockk<GooglePayLauncherCallback>(relaxed = true)
    private val activityResultRegistry = mockk<ActivityResultRegistry>(relaxed = true)
```

In Kotlin the type can actually be inferred. So you can either declare the type and not include the type in the `mockk()` constructor, or remove the declared type and specify the type in the mockk constructor.

Reply:
Ah yes! That looks much better. Thanks for the info!